src/core/mac/mac_frame.cpp

```diff
@@ -116,6 +116,12 @@ void TxFrame::Info::PrepareHeadersIn(TxFrame &aTxFrame) const
         // | 13 | Extended     | Short        | Present      | Not Present  |      1       |
         // | 14 | Short        | Short        | Present      | Not Present  |      1       |
         // +----+--------------+--------------+--------------+--------------+--------------+
+        //
+        // This table shows the combination of flags allowed in an encoded MAC
+        // header. Regarding rows 9-14, when both Source and Destination
+        // Address fields are present and at least one uses a short address
+        // format, then if the source and destination PAN IDs are equal, PAN
```

Comment:
For `if the source and destination PAN IDs are equal`, if the `Src PAN ID` is not present, we think the `Src PAN ID` and `Dst PAN ID` are the same, right?

Reply:
Yes. This is how it should be interpreted when decoding the frame.